packages/rgbpp/src/ckb.ts

```diff
@@ -0,0 +1 @@
+export * from '@rgbpp-sdk/service';
```

Comment:
The file name seems to be inconsistent with the exported package

Reply:
Fixed